### PR TITLE
Add generate_media_config to robotInterface

### PR DESCRIPTION
**Pull request by Eddasol**
To match breaking change in ISAR to be introduced in this PR: https://github.com/equinor/isar/pull/667

### CODE REVIEW

File: src/isar_robot/robotinterface.py [hunk 1]
```diff
@@ -175,3 +176,6 @@ def pause(self) -> None:
 
     def resume(self) -> None:
         return
+
+    def generate_media_config(self) -> MediaConfig:
+        return
```
Comment (sondreo) [inline src/isar_robot/robotinterface.py:181]: Should we return a default `MediaConfig` instance to match the function type-hinting?

Comment (Eddasol) [reply src/isar_robot/robotinterface.py:181]: Isn't it better to send None than som random value? It makes it easier for isar  to know that there is no valid mediaConfig

Comment (sondreo) [reply src/isar_robot/robotinterface.py:181]: Raising an exception could be a good idea!

Comment (Eddasol) [reply src/isar_robot/robotinterface.py]: Will change return type to optional both here and in isar

**Review by tsundvoll — APPROVED** [review]

**Pull request merged**